tests/test_cadquery.py

```diff
@@ -4177,3 +4216,42 @@ def testEnd(self):
         self.assertTrue(Workplane().objects == [])
         self.assertTrue(Workplane().box(1, 1, 1).end().objects == [])
         self.assertTrue(Workplane().box(1, 1, 1).box(2, 2, 1).end(2).objects == [])
+
+    def testCutEach(self):
+
+        # base shape:
+        w = Workplane().box(3, 2, 2)
+        # cutter:
+        c = Workplane().box(2, 2, 2).val()
+        # cut all the corners off
+        w0 = w.vertices().cutEach(lambda loc: c.located(loc))
+        # we are left with a 1x2x2 box:
+        self.assertAlmostEqual(w0.val().Volume(), 4, 3)
+
+        # test error on no solid found
+        w1 = Workplane().hLine(1).vLine(1).close()
+        with raises(ValueError):
+            w1.cutEach(lambda loc: c.located(loc))
+
+    def testCutBlind(self):
+        # cutBlind is already tested in several of the complicated tests, so this method is short.
+        # test ValueError on no solid found
+        w0 = Workplane().hLine(1).vLine(1).close()
+        with raises(ValueError):
+            w0.cutBlind(1)
+
+    def testFindFace(self):
```

Comment:
Do you still want to add this test based on the decision in #665 ?

Reply:
Yeah, might as well keep the test while `findFace` is marked deprecated, then remove the test along with `findFace` in the next release.